app/partials/collection-card.cjsx

```diff
@@ -64,6 +64,7 @@ module.exports = React.createClass
         <svg className="card-space-maker" viewBox="0 0 2 1" width="100%"></svg>
         <div className="details">
           <div className="name"><span>{@props.collection.display_name}</span></div>
+          {<div className="public-spaceholder">&nbsp;</div> if !@props.collection.private}
```

Comment:
ok. it was just a quick hack to fix an alignment issue i noticed. Not really relevant to this PR so will remove it for now.